### PR TITLE
Updated registration files.

### PR DESCRIPTION
**Pull request by SoroorAghamiri**
Updated registration code by moving error checking to html file, and removing checking user by username and password. Added Styles.css (to use, include "/Design/styles.css" in your html header tag.)
Documentation added.
#15

### CODE REVIEW

**Review by sepehr9877 — APPROVED** [review]

File: src/main/java/ca/concordia/soen6011/controller/RegistrationController.java [hunk 1]
```diff
@@ -100,13 +116,9 @@ private void registerCandidate(String username, String email, String firstName,
 
         candidate.setPassword(password);
         //set user type
+        try{
```
Comment (MajorTreble) [inline src/main/java/ca/concordia/soen6011/controller/RegistrationController.java:119]: The alignment is off

File: src/main/java/ca/concordia/soen6011/controller/RegistrationController.java [hunk 2]
```diff
@@ -86,12 +93,21 @@ private void registerEmployer(String username, String email, String firstName, S
 
         employer.setPassword(password);
         //set user type
-        employerRepository.save(employer);
-
+        try {
+            employerRepository.save(employer);
+        }catch (Exception e){}
```
Comment (MajorTreble) [inline src/main/java/ca/concordia/soen6011/controller/RegistrationController.java:98]: catch on a new line

File: src/main/java/ca/concordia/soen6011/controller/RegistrationController.java [hunk 3]
```diff
@@ -8,13 +8,19 @@
 import ca.concordia.soen6011.repository.UserRepository;
 import jakarta.servlet.http.HttpSession;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
```
Comment (MajorTreble) [inline src/main/java/ca/concordia/soen6011/controller/RegistrationController.java:11]: Are you using this?

**Review by MajorTreble — APPROVED** [review]

File: src/main/java/ca/concordia/soen6011/controller/RegistrationController.java (hunk 3 above)
Comment (SoroorAghamiri) [reply src/main/java/ca/concordia/soen6011/controller/RegistrationController.java:11]: yes, I'm getting a request from html file to check the user name, I process and respond to that request using ResponseEntity

**Pull request merged**